proof/crefine/RISCV64/Invoke_C.thy

```diff
@@ -260,7 +281,8 @@ done
 
 lemma invokeCNodeCancelBadgedSends_ccorres:
   "ccorres (cintr \<currency> dc) (liftxf errstate id (K ()) ret__unsigned_long_')
-     (invs' and sch_act_simple and valid_cap' cap and K (isEndpointCap cap)) (UNIV \<inter> {s. ccap_relation cap (cap_' s)}) []
+     (invs' and sch_act_simple and valid_cap' cap and K (isEndpointCap cap))
+     (UNIV \<inter> {s. ccap_relation cap (cap_' s)}) []
```

Comment:
Do we want to remove the `UNIV` from lemmas that are already being modified?

Reply:
I'd vote for removing it if we touch the lemma.